docs/001_develop/03_client-capabilities/008_interaction/checkbox.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: 'Checkbox'
+sidebar_label: 'Checkbox'
+id: client-interaction-checkbox
+keywords: [interaction, interactive, checkbox, box, boolean, form, click, change, checked, toggle, submit, data]
+tags:
+- interaction
+- interactive
+- checkbox
+- box
+- boolean
+- form
+- click
+- change
+- checked
+- toggle
+- submit
+- data
+sidebar_position: 8
+---
+
+import CheckboxDemo from '../../../../examples/ui/client-capabilities/interaction/checkbox.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Checkbox
+
+A form-connected implementation of a [checkbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox) input.
+
+## Example
+
+<CheckboxDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+<TabItem value="genesis">
+
+Declaration
+```typescript
+<rapid-checkbox></rapid-checkbox>
+```
+Usage
+```typescript
+@customElement({
+    name: 'my-element',
+    templte: html`
+        <rapid-checkbox
+            ?disabled="${(x) => x.disabled}"
+            ?checked="${sync((x) => x.isSelected, 'boolean')}"
+            @change="${(x) => x.checkChanged()}"
+        >Checkbox
+        </rapid-checkbox>
+    `,
+})
+export class MyElement extends GenesisElement {
+    @observable disabled = false;
+    @observable isSelected = false;
+    checkChanged() {
+        console.log(this.isSelected);
+    }
+}
+```
+
+</TabItem>
+<TabItem value="react">
+
+Declaration
+```typescript
+<rapid-checkbox></rapid-checkbox>
+```
+Usage
+```typescript
+export function MyComponent() {
+    const [isSelected, setIsSelected] = useState(false);
+    // TODO: import this listener!
```

Comment:
TODO left in place

Reply:
Yeah that was intentionally left in because the react code needed updating to align with the latest syntax. I have updated the whole example here bfe13d09b61cc7d399b8fdf56192e338e79ee2fb